src/inference/src/ie_core.cpp

```diff
@@ -899,40 +899,6 @@ class CoreImpl : public ie::ICore, public std::enable_shared_from_this<ie::ICore
         OV_ITT_SCOPED_TASK(ov::itt::domains::IE, "Core::QueryNetwork");
         auto parsed = parseDeviceNameIntoConfig(deviceName, config);
         auto res = GetCPPPluginByName(parsed._deviceName).query_model(network, parsed._config);
-        if (!network.getFunction() || res.supportedLayersMap.empty())
-            return res;
-
-        const auto& func = network.getFunction();
-        auto specialized_function = ngraph::clone_function(*func);
-
-        std::string defDevice = res.supportedLayersMap.begin()->second;
-        ngraph::pass::ConstantFolding().run_on_model(specialized_function);
-        std::unordered_set<std::string> opNames;
-
-        for (const auto& op : specialized_function->get_ops())
-            opNames.emplace(op->get_friendly_name());
-
-        for (const auto& op : func->get_ops()) {
-            if (opNames.find(op->get_friendly_name()) == opNames.end()) {
-                res.supportedLayersMap[op->get_friendly_name()] = defDevice;
-            }
-        }
-
-        for (const auto& op : func->get_ops()) {
-            if (!res.supportedLayersMap.count(op->get_friendly_name()) &&
-                std::dynamic_pointer_cast<ngraph::op::Constant>(op)) {
-                bool are_all_users_supported = true;
-                for (const auto& user : op->output(0).get_target_inputs()) {
-                    if (!res.supportedLayersMap.count(user.get_node()->get_friendly_name())) {
-                        are_all_users_supported = false;
-                        break;
-                    }
-                }
-                if (are_all_users_supported) {
-                    res.supportedLayersMap[op->get_friendly_name()] = defDevice;
-                }
-            }
-        }
```

Comment:
it's already removed here https://github.com/openvinotoolkit/openvino/pull/15315, maybe just need to close the PR?